docs/output.md

```diff
@@ -21,51 +17,67 @@ The directories listed below will be created in the output directory after the p
-* `guppy/basecalling/guppy_basecaller_log-<date>.log`  
-  Log file for *Guppy* execution.
-* `qcat/fastq/<barcode*>.fastq.gz`  
-  fastq output files for each barcode.
-* `qcat/fastq/none.fastq.gz`  
-  fastq file with reads were unassigned to any given barcode.
+* `guppy/fastq/`
+    Merged fastq output files for each barcode.
+* `guppy/basecalling/<barcode*>/`
+    fastq output files for each barcode.
+* `guppy/basecalling/unclassified/`
+    fastq files with reads were unassigned to any given barcode.
+* `guppy/basecalling/sequencing_summary.txt`
+    Sequencing summary file generated by *Guppy*.
+* `guppy/basecalling/sequencing_telemetry.js`
+    Sequencing telemetry file generated by *Guppy*.
+* `guppy/basecalling/guppy_basecaller_log-<date>.log`
+    Log file for *Guppy* execution.
+* `qcat/fastq/<barcode*>.fastq.gz`
+    fastq output files for each barcode.
+* `qcat/fastq/none.fastq.gz`
+    fastq file with reads were unassigned to any given barcode.
 
 </details>
 
-*Documentation*:  
+*Documentation*:
 [Guppy](https://nanoporetech.com/nanopore-sequencing-data-analysis), [qcat](https://github.com/nanoporetech/qcat)
 
-*Description*:  
+*Description*:
 The pipeline has been written to deal with the various scenarios where you would like to include/exclude the basecalling and demultiplexing steps. This will be dependent on what type of input data you would like to provide the pipeline. Additionally, if you would like to align your samples to a reference genome there are various options for providing this information. Please see [`usage.md`](usage.md#--input) for more details about the format of the input samplesheet, associated commands and how to provide reference genome data.
 
 *Guppy* will be used to basecall and demultiplex the data. Various options have been provided to customise specific parameters and to be able to run *Guppy* on GPUs.
 
 If you have a pre-basecalled fastq file then *qcat* will be used to perform the demultiplexing if you provide the `--skip_basecalling` parameter. If you would like to skip both of these steps entirely then you can provide `--skip_basecalling --skip_demultiplexing` when running the pipeline. As a result, the structure of the output folder will depend on which steps you have chosen to run in the pipeline.
 
+## Removal of DNA contaminants
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `nanolyse/<SAMPLE>.clean.fastq.gz`
+    FastQ file after the removal of reads mapping to DNA contaminants.
+
+</details>
+
+*Documentation*:
+[NanoLyse](https://github.com/wdecoster/nanolyse)
+
+*Description*:
+If you would like to run NanoLyse on the raw FastQ files then you can provide `--run_nanolyse` when running the pipeline. By default, the pipeline will filter the raw reads relative to lambda phage but you can provide your own fasta file of "contaminants" with `--nanolyse_fasta`. The filtered FastQ files will be used in the downstream processes of the pipeline.
```

Comment:
This reads more as 'usage' information rather than output? Maybe you could rephrase to just say what the filtered FASTQ will contain if you run X vs Z.

Reply:
I have changed the description to "The filtered FastQ files will contain raw reads without the provided reference sequences (default: lambda phage sequences)."

Reply:
I was mostly referring to the earlier sentences, but this OK for now.